crates/daphne-server/src/storage_proxy_connection/kv/cache.rs

```diff
@@ -30,12 +30,9 @@ pub struct Cache {
 }
 
 pub enum GetResult<T: 'static> {
-    NotFound {
-        /// Whether the caller should check KV for the value.
-        read_through: bool,
-    },
+    NotFound,
     MismatchedType,
-    Found(Marc<T>),
+    Found(Option<Marc<T>>),
```

Comment:
Is this what you intend?
```suggestion
    /// None indicates no value was found, but that the caller should cache the negative response.
    Found(Option<Marc<T>>),
```